contracts/interfaces/ICryptoBlades.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.6.5;
+
+interface ICryptoBlades {
```

Comment:
Make the CryptoBlades contract properly implement the interface, and mark functions as overridden as needed by the compiler.